teuthology/task/install/__init__.py

```diff
@@ -232,8 +232,12 @@ def install(ctx, config):
     # pull any additional packages out of config
     extra_pkgs = config.get('extra_packages')
     log.info('extra packages: {packages}'.format(packages=extra_pkgs))
-    debs += extra_pkgs
-    rpm += extra_pkgs
+    if isinstance(extra_pkgs, dict):
+        debs += extra_pkgs.get('deb')
```

Comment:
perhaps .get(type, []) instead so that missing one type doesn't result in an exception trying to add None to a list